### ISSUE TITLE
Roadmap: Markdown

### ISSUE THREAD

**Issue by marcobiedermann** [stevenvachon/broken-link-checker#35]
> option to parse Markdown in HtmlChecker for links

You could use a markdown parser like [marked](https://www.npmjs.com/package/marked) to work with Markdown files.

```
var marked = require('marked');

htmlChecker.scan(marked('I am using __markdown__.'));
```

**Comment by stevenvachon** [comment]
Indeed. I haven't started solving it yet, but it would need to be a dependency so that accurate location info can be provided. The goal is to provide the tools for a front-end that can correct the original source.
